### DISCUSSION TITLE
Theme query and result item fonts working?

### DISCUSSION THREAD

**Discussion by jjw24** [Flow-Launcher/Flow.Launcher#625] [Q&A]
is it me or does changing this options not working fully? the preview doesnt seem to refresh and seems like font changes works after a restart, but not all font options are working.

![image](https://user-images.githubusercontent.com/26427004/127785534-31ca0515-3809-47fa-8883-e1b2bc351296.png)

**Comment by taooceros** [comment]
I also encounter the issue. Theme need to be changed to another and changed back to see the font update.

**Reply by mmelvin0** [reply]
Also does not seem to apply at all if "Query window shadow effect is turned on"

**Reply by jjw24** [reply]
This issue is resolved and fix is available in pre-release.
